hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconDeadNodeHandler.java

```diff
@@ -71,6 +77,8 @@ public void onMessage(final DatanodeDetails datanodeDetails,
         LOG.warn("Node {} has reached DEAD state, but SCM does not have " +
             "information about it.", datanodeDetails);
       }
+      containerHealthTask.triggerContainerHealthCheck();
+      pipelineSyncTask.triggerPipelineSyncTask();
```

Comment:
Q: Would a single `triggerContainerHealthCheck` or `triggerPipelineSyncTask` run be time consuming or resource intensive? Since this imply every time `onMessage` here is entered, those two tasks would be triggered.

Reply:
@smengcl - thanks for review. I understood your question, so here is explanation:
Both the existing tasks "`ContainerHealthTask`" and "`PipelineSyncTask`" are currently being run at interval of 5 mins and they are not resource intensive, so after refactoring the code and moved the logic to "`triggerContainerHealthCheck`" its being called on DeadNode event which is configured default as 10 mins and for "`triggerPipelineSyncTask`", this is being called on both DeadNode (Configured default interval is 10 mins) and StaleNode event (Configured default interval is 5 mins), so there will not be any issue as existing also these tasks were run each at interval of 5 mins.